.pipelines/templates/unit-test.yaml

```diff
@@ -4,10 +4,9 @@ jobs:
   cancelTimeoutInMinutes: 5
   workspace:
     clean: all
+  variables:
+    - group: csi-secrets-store-e2e-kind
```

Comment:
Why do we need this variable group for unit tests?

Reply:
We need `TRIVY_VERSION` for scan-images job.

Reply:
That can be set as a pipeline variable

Reply:
makes sense. Updated.